docs/FAQ.md

```diff
@@ -157,12 +157,22 @@ Usually, the different commits associated with the conflicted branch should all
 appear in the log, but if they don't you can use `jj branch list`to show all the
 commits associated with it.
 
+### How do I integrate Jujutsu with Gerrit? 
+
+At the moment you'll need a script, which adds the required fields for Gerrit
+like the `Change-Id` footer. Then `jj` can invoke it via an `$EDITOR` override
+in an aliased command. Here's an [example][gerrit-integration] from an 
+contributor (look for the `jj signoff` alias).
```

Comment:
I wonder if we shouldn't just include the `signoff.sh` script in this repository as an example? I'm not really sure. I have tweaked mine a few times &mdash; including using a 20 byte Change-ID string (40 hex characters) &mdash; and haven't updated the gist. So maybe it should belong here. I'm somewhat neutral on that.

Reply:
> I wonder if we shouldn't just include the `signoff.sh` script in this repository as an example?

For the initial version your script is ok, but we should move a simplified version of it to the wiki. But this can be done in a follow-up.